flume/conf/README.md

```diff
@@ -0,0 +1,54 @@
+# Cygnus configuration Procedure
+
+From version 0.6.0 Cygnus can start multiple instances by renaming and configuring some of 
+templates files of this directory.
+
+There are two types of configuration template files:
+1- Files which is only one for all intances
+2- Files wich there are one per each intance that is wanted to run.
+
+## 1- Unique files for all instances
+
+For the first group files it is necessary rename (or better copy) it without `.template` extension.
+These files are: `log4j.properties.template`, `krb5.conf.template` and `matching_table.conf.template`
+
+```bash
+cp /usr/cygnus/log4j.properties.template /usr/cygnus/log4j.properties
+cp /usr/cygnus/krb5.conf.template /usr/cygnus/krb5.conf
+cp /usr/cygnus/matching_table.conf.template /usr/cygnus/matching_table.conf
+```
+
+Then each file can be edited to changed configuration that will affect to all intances of cygnus.
+
+## 2- Files for each instance
+
+The other files are `cygnus.conf.template` and `flume.conf.template`. This file should be copied 
```

Comment:
Once I've seen how the two configuration files work, I think it is better to rename the template files as:

cygnus.conf.template --> agent.conf_template (because this file contains the Flume agent configuration of Cygnus)
flume.conf.template --> cygnus.conf.template (because this file contains other non-Flume configuration of Cygnus)

I.e. a file